chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -357,7 +359,23 @@
     --is-public True \
     --page-size 1 \
     2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
-[ -n "$DOCKER_IMAGE_ID" ] && echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+if [ -n "$DOCKER_IMAGE_ID" ] ; then
+  echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+else
+  for node in #{docker_nodes.join(" ")}; do
+      ssh $node docker pull cirros
```

Comment:
Nope, that's not enough; there's an explicit limitation that image must be also in the local docker repo of each node right now (which is annoying, indeed).